src/components/toast/toast.js

```diff
@@ -32,7 +32,7 @@ export const props = {
   },
   variant: {
     type: String,
-    default: null
+    default: () => getComponentConfig(NAME, 'variant') || null
```

Comment:
I think we should omit the fallbacks, since we don't do that for the other components.

Reply:
True...